FakebookNotifications/FakebookNotifications.DataAccess/Models/Notification.cs

```diff
@@ -0,0 +1,34 @@
+﻿using MongoDB.Bson;
```

Comment:
if this code was going to be unit tested, it'd need to have `get => throw new NotImplementedException()` for all these so that the tests fail. but classes like this with only auto-properties don't need to be unit tested.
(hence these classes should be added in an unlabelled PR, not this PR, but i'll overlook that)